projects/plugins/boost/app/assets/src/js/pages/settings/sections/Score.svelte

```diff
@@ -31,9 +36,12 @@
 				await clearCache();
 			}
 
-			scores = await requestSpeedScores();
+			let scoresSet = await requestSpeedScores();
+			scores = scoresSet.current;
+			previousScores = scoresSet.previous;
```

Comment:
Be sure to use `const` for any variable you don't intend to change, for clarity.